modules/gapi/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ set(gapi_srcs
     src/backends/cpu/gcpuimgproc.cpp
     src/backends/cpu/gcpuvideo.cpp
     src/backends/cpu/gcpucore.cpp
+    src/backends/cpu/gparsers.cpp
```

Comment:
Since these parsers are not generic but mainly for NN output, I'd rename all these files to `gnnparsers.*` - not to confuse anybody.

Reply:
Fixed.